sw.js

```diff
@@ -1,103 +1,47 @@
 importScripts(
-  'js/vendor/sw-cacheable-response.min.js',
-  'js/vendor/sw-routing.min.js',
-  'js/vendor/sw-runtime-caching.min.js',
-  'js/vendor/sw-precaching.min.js'
+  'https://storage.googleapis.com/workbox-cdn/releases/3.3.1/workbox-sw.js'
 );
 
-// Make this always match package.json version
-const version = '0.1.3';
-const cacheName = `defaultCache_${version}`;
-
-const { cacheableResponse, precaching, routing, runtimeCaching } = goog;
-
-const router = new routing.Router();
-const localhost = registration.scope;
-const staticCache = new precaching.UnrevisionedCacheManager({ cacheName });
-const requestWrapper = new runtimeCaching.RequestWrapper({ cacheName });
-const cdnRequestWrapper = new runtimeCaching.RequestWrapper({
-  plugins: [
-    new cacheableResponse.CacheableResponsePlugin({ statuses: [0, 200] })
-  ],
-  cacheName
-});
-
-/**
- * Route for local CSS and JS assets
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#stale-while-revalidate
- */
-const assetRoute = new routing.RegExpRoute({
-  regExp: new RegExp(`^${localhost}.*\\.(css|js|png|svg)$`),
-  handler: new runtimeCaching.StaleWhileRevalidate({ requestWrapper })
-});
+// Cache fallback for HTML files
+workbox.routing.registerRoute(
+  /(?:^[^.]+|\.html)$/,
+  workbox.strategies.networkFirst()
+);
 
-/**
- * Route for assets on cloudfront.net CDN
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#stale-while-revalidate
- */
-const cdnAssetRoute = new routing.RegExpRoute({
-  regExp: new RegExp('^https://.*\\.cloudfront\\.net/.*\\.(png|svg)$'),
-  handler: new runtimeCaching.StaleWhileRevalidate({
-    requestWrapper: cdnRequestWrapper
+// Cache CSS and JS files
+workbox.routing.registerRoute(
+  /\.(?:js|css)$/,
+  workbox.strategies.staleWhileRevalidate({
```

Comment:
For static assets, we should use a [`cacheFirst()`](https://developers.google.com/web/tools/workbox/modules/workbox-strategies#cache_first_cache_falling_back_to_network) strategy instead. 😉

Reply:
Can you explain why? My reasoning for using [`staleWhileRevalidate`](https://developers.google.com/web/tools/workbox/modules/workbox-strategies#stale-while-revalidate) is that then it would update the file in the background (for example if the css file is updated)

Reply:
@calebeby You've convinced me. I did some more digging and it looks like using `staleWhileRevalidate` is a [common way to handle CSS & JS that isn't precached](https://developers.google.com/web/tools/workbox/guides/common-recipes#cache_css_and_javascript_files). Thanks for double-checking! 😉